### ISSUE TITLE
Moving cmbr to github.molgen.mpg.de

### ISSUE THREAD

**Issue by your-highness** [matthuska/cmbr#1]
What do you think about moving this repository to our institute github server?

https://github.molgen.mpg.de/

**Comment by matthuska** [comment]
Yes I think that's a good idea. It will mean that people will need to update any repositories that they have cloned though. We should make sure that everyone who is using it knows (Ale, you, me and maybe Matthias I think).

Just fyi, there are instructions for moving code from regular github to github enterprise here:

https://help.github.com/enterprise/2.0/admin/articles/moving-a-repository-from-github-com-to-github-enterprise/

**Comment by matthuska** [comment]
I've moved the package over to the MPI's internal GitHub enterprise instance:

https://github.molgen.mpg.de/matthuska/cmbr

I also took the opportunity to convert the code to a proper R package, which can be installed like any other git-based R package using devtools.

Right now a lot of functions were causing problems so most of the code is commented out except for a few functions that I need right now. I'm trying to be a bit more selective about making sure that all of the functions are documented as well, now that we can actually easily use that documentation (?bed2GR for instance). Feel free to fork and send pull requests if you want to contribute. Future discussion can take place at the above url.
